dff/utils/testing/common.py

```diff
@@ -8,6 +8,12 @@
 
 
 def is_interactive_mode() -> bool:
+    """
+    Checking whether the example code should be run in interactive mode.
+
+    :return: true only if it's being executed by Jupyter kernel AND DISABLE_INTERACTIVE_MODE env variable isn't set.
```

Comment:
I don't see how this is true.

The function returns true if:

1. `IPython` module is not installed
2. shell is not `ZMQInteractiveShell`

It seems the function returns false only if it's being executed by Jupyter kernel.

Reply:
Generally you are right, it returns false only if it's being executed by Jupyter kernel and environmental variable is not set. That means acually that the example code shouldn't be run in interactive mode.
What's wrong with it?

Reply:
The doc reads `:return: true only if it's being executed by Jupyter kernel`.